packages/@yodaos/step/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.0)
+project(yodaos-step CXX)
+set(CMAKE_CXX_STANDARD 11)
+
+file(GLOB MEDIAKIT_SRC *.js)
```

Comment:
Variable name not changed.

Also, should we extract these cmake scripts into a common shared file?

Reply:
Agree. I think use a cmake file to install all packages will be better. So we don't have to write cmake files for each package.